### ISSUE TITLE
Python libraries licenses are not gathered

### ISSUE THREAD

**Issue by templier2** [anchore/syft#2970]
**What happened**:
syft shows python packages, but doesn't collect licenses.

**What you expected to happen**:
I want to see licenses for packages

**Steps to reproduce the issue**:
I run
anchore_syft packages . -o cyclonedx-xml=bom.xml

**Anything else we need to know?**:
unsure

**Environment**:
- Output of `syft version`:
```
Application:        syft
Version:            0.76.0
JsonSchemaVersion:  7.0.1
BuildDate:          2023-03-31T15:11:44Z
GitCommit:          dfcc07e5122217ca9e2fc75817c593356fc0c405
GitDescription:     v0.76.0
Platform:           linux/amd64
GoVersion:          go1.19.7
Compiler:           gc
```
- OS (e.g: `cat /etc/os-release` or similar):
```
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

**Comment by kzantow** [comment]
Hi @templier2, what type of python packages are you scanning? It looks like today Syft is only able to find licenses from wheel/egg files, and perhaps the license information isn't present in other packaging formats? Could you provide a reproducible case, such as a public docker image we could look at to help understand what's there and let us know what you expect?

**Comment by templier2** [comment]
> Hi @templier2, what type of python packages are you scanning? It looks like today Syft is only able to find licenses from wheel/egg files, and perhaps the license information isn't present in other packaging formats? Could you provide a reproducible case, such as a public docker image we could look at to help understand what's there and let us know what you expect?

Hi,
It is a list of libraries from requirements.txt, I suppose that every library has appropriate information about the license at pypi.org (i.e. https://pypi.org/project/rdflib/)
databricks-connect==14.3.2
rdflib==7.0.0
SPARQLWrapper==2.0.0
azure-search-documents==11.4.0
pyspark==3.5.1
lxml==5.2.2

**Comment by wagoodman** [comment]
Ahh I see, so that is being scanned is purely the source code directory (finding the requirements.txt?) and not against installed packages. 

```
```

**Comment by wagoodman** [comment]
I'm going to recategorize this is a feature request, since it sounds like what you're asking for is for syft to be able to remotely gather information about the package declared locally against package registries and enrich the SBOM. Is this accurate @templier2 ?

**Comment by templier2** [comment]
> I'm going to recategorize this is a feature request, since it sounds like what you're asking for is for syft to be able to remotely gather information about the package declared locally against package registries and enrich the SBOM. Is this accurate @templier2 ?

Yes, but it seems that your option with installing it within a venv environment is ok for us too.
Thank you.

**Comment by elaine-mattos** [comment]
> I'm going to recategorize this is a feature request, since it sounds like what you're asking for is for syft to be able to remotely gather information about the package declared locally against package registries and enrich the SBOM. Is this accurate [@templier2](https://github.com/templier2) ?

Hi, is it possible to have this as a feature request? It would be of great help to us!
